contracts/router/AedexV2Router.aes

```diff
@@ -93,21 +97,25 @@ payable contract AedexV2Router =
                 amount_a_min, amount_b_min,
                 min_liquidity
                 )
-        //getting pair and transfer to its address the tokens
+        //getting pair and transfer the tokens to its address
         let pair: IAedexV2Pair = AedexV2Library.pair_for( state.factory, token_a, token_b )
         token_a.transfer_allowance( Call.caller,pair.address,amount_a )
         token_b.transfer_allowance( Call.caller,pair.address,amount_b )
         let minted_liquidity = pair.mint(to)
         ( amount_a, amount_b, minted_liquidity )
 
+    // Adds liquidity for a certain Pair of a regular AEX9 token and AE.
```

Comment:
Perhaps, you could just refer to the docs of `add_liquidity` and mention how `add_liquidity_ae` differs instead of duplicating the comment.